apps/browser-tests/src/main.ts

```diff
@@ -1,16 +1,4 @@
 import { unisat } from "@ordzaar/ordit-sdk/browser-wallets";
 
-async function main() {
-  const appRoot = document.getElementById("app");
-
-  if (!appRoot) {
-    return;
-  }
-
-  appRoot.innerHTML = "Browser Test";
-
-  const isUnisatInstalled = await unisat.isInstalled();
-  console.log("Is Unisat Installed:", isUnisatInstalled);
-}
-
-main();
+const isUnisatInstalled = unisat.isInstalled();
```

Comment:
We can add a button to call this. Even just a simple one, so we can test out installation.

Reply:
I actually added that in the next PR, but will move it here instead.